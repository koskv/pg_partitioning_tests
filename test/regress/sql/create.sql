CREATE TABLE test_cc (
    id serial,
    dt date
);

CREATE FUNCTION partition_insert_trigger() RETURNS trigger
    LANGUAGE c STRICT
    AS 'partition_insert_trigger', 'partition_insert_trigger'
	SET DateStyle TO 'ISO,MDY';

