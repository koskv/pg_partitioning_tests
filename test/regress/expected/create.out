CREATE TABLE test_cc (
    id serial,
    dt date
);
NOTICE:  CREATE TABLE will create implicit sequence "test_cc_id_seq" for serial column "test_cc.id"
CREATE FUNCTION partition_insert_trigger() RETURNS trigger
    LANGUAGE c STRICT
    AS 'partition_insert_trigger', 'partition_insert_trigger'
	SET DateStyle TO 'ISO,MDY';
